components/ActivityIndicator/src/private/MDCActivityIndicatorMotionSpec.m

```diff
@@ -16,51 +16,71 @@
 
 #import "MDCActivityIndicatorMotionSpec.h"
 
-const NSTimeInterval kPointCycleDuration = 4.0f / 3.0f;
-const NSTimeInterval kPointCycleMinimumVariableDuration = kPointCycleDuration / 8;
+@implementation MDCActivityIndicatorMotionSpec
 
-const struct MDCActivityIndicatorMotionSpec kMDCActivityIndicatorMotionSpec = {
-  .indeterminate = {
++ (NSTimeInterval)pointCycleDuration {
+  return 4.0f / 3.0f;
+}
+
++ (NSTimeInterval)pointCycleMinimumVariableDuration {
+  return [self pointCycleDuration] / 8;
+}
+
++ (MDCActivityIndicatorMotionSpecIndeterminate)loopIndeterminate {
+  NSTimeInterval pointCycleDuration = [self pointCycleDuration];
```

Comment:
Super-Nit: Use class name (instead of `self`) unless you want subclasses able to override.

Reply:
Done.